setup.php

```diff
@@ -241,3 +243,10 @@ function plugin_datainjection_checkDirectories()
     }
     return true;
 }
+
+function plugin_datainjection_geturl(): string
+{
+    /** @var array $CFG_GLPI */
+    global $CFG_GLPI;
+    return sprintf('%s/plugins/datainjection/', $CFG_GLPI['url_base']);
+}
```

Comment:
I'm not a fan of using a function in this case, because it doesn't make the code easier to read.
What's more, in some cases, you have to use `$CFG_GLPI[‘root_doc’]` instead of `$CFG_GLPI[‘url_base’]`.